server/src/main/java/org/apache/druid/segment/handoff/CoordinatorBasedSegmentHandoffNotifier.java

```diff
@@ -114,10 +115,10 @@ void checkForSegmentHandoffs()
         catch (Exception e) {
           log.error(
               e,
-              "Exception while checking handoff for dataSource[%s] Segment[%s], Will try again after [%d]secs",
+              "Exception while checking handoff for dataSource[%s] Segment[%s]; will try again after [%s]",
               dataSource,
               descriptor,
-              pollDurationMillis
+              pollDuration.toString()
```

Comment:
You can just use `pollDuration` instead of `pollDuration.toString()` here as the `.toString()` is implicit.